test/Microsoft.AspNetCore.Mvc.Analyzers.Test/SymbolApiConventionMatcherTest.cs

```diff
@@ -0,0 +1,568 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Analyzer.Testing;
+using Microsoft.AspNetCore.Mvc.Analyzers.Infrastructure;
+using Microsoft.CodeAnalysis;
+using Xunit;
+using static Microsoft.AspNetCore.Mvc.Analyzers.SymbolApiConventionMatcher;
+
+namespace Microsoft.AspNetCore.Mvc.Analyzers
+{
+    public class SymbolApiConventionMatcherTest
+    {
+        private static readonly string BaseTypeName = typeof(Base).FullName;
```

Comment:
These tests are near identical to the `ApiConventionMatcherTest` (same types etc but using symbols instead of reflected types)